### DISCUSSION TITLE
Getting issues on creating project.

### DISCUSSION THREAD

**Discussion by jswapneswar** [implerhq/impler.io#623] [Q&A]
I deployed impler using Docker Compose, and everything running fine. I am able to signup and login but after that, I am unable to create project and getting 401 unathorized error. Any solution for this?

**Comment by chavda-bhavik** [comment]
Hey @jswapneswar Can you share more details along with Screenshots? Like the photo of the network tab calling API. Also CURL request throwing 401. It will help check the issue. Thanks!
